images/manageiq-base/Dockerfile

```diff
@@ -45,18 +47,32 @@ RUN yum -y install https://rpm.nodesource.com/pub_10.x/el/7/x86_64/nodesource-re
                    lvm2             \
                    net-tools        \
                    nmap-ncat        \
-                   nodejs           \
                    openldap-clients \
                    openscap-scanner \
                    patch            \
                    psmisc           \
-                   postgresql-devel \
                    sqlite-devel     \
+                   postgresql-devel \
```

Comment:
Why was this changed here?

Reply:
Changes atop each other caused a sequence change, sorry about it, will correct